docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,59 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a {doc}`native Azure implementation
+</object-storage/file-system-azure>` that does not rely on the legacy Hadoop
+implementation. Trino recommends upgrading existing deployments to the new
+native implementation at your earliest convenience.
+
+The following considerations apply to legacy configurations moving to the
+native implementation:
+
+* Legacy storage types (WASB and ADL) are not supported.
+* Azure Government is not supported.
+
+To migrate a catalog to use the native filesystem implementation for Azure, make
+the following edits to your catalog configuration:
+
+1. Add the `fs.native-azure.enabled=true` catalog configuration property.
+2. Add the `azure.auth-type` catalog configuration property. Set to `ACCESS_KEY`
```

Comment:
Just say "Configure the `azure.auth-type` catalog configuration property." 

Dont worry about the details .. its documented on the same page above. If you really want you can link to it

Reply:
I disagree. Migration guides should have complete info and not require clicking around if at all possible.

Reply:
The problem is that this will get super complex with S3 and potentially any others like minio usage. We would need to at least cover required properties I think. Also .. it has to be maintained as duplicate info from now on until we remove the duplication (or the whole migration section) .. not sure if this is worth the effort.